src/software/SfM/main_ComputeMatches.cpp

```diff
@@ -126,6 +126,7 @@ int main(int argc, char **argv)
       << "  AUTO: auto choice from regions type,\n"
       << "  For Scalar based regions descriptor:\n"
       << "    BRUTEFORCEL2: L2 BruteForce matching,\n"
+      << "    HNSWL2: L2 Approximate Matching with Hierarchical Navigable Small World graphs (float only),\n"
```

Comment:
I'm removing float only since you have enabled uchar as a type too

Reply:
yes was an artifact of my test implementation, sorry.

Reply:
No worries. I was just double-checking!